webots_ros2_core/webots_ros2_core/publishers/publisher.py

```diff
@@ -0,0 +1,6 @@
+class Publisher:
+    def publish(self):
+        pass
```

Comment:
Probably we can add an assert here to be sure this is overridden in the derived class.